### PR TITLE
Tatari Typeahead Upgrades & CSS Cleanup

### PR DESCRIPTION
**Pull request by nickwmurphy**
1. The CSS file was alphabetized, and I cleaned up unnecessary overrides.

2. Many small style tweaks were made to center things, and the dropdown body is now responsive with a min- and max-width.

3. The sandbox file has "more realistic" data now.

4. The typeahead search bar in the dropdown body now has a placeholder value.

5. Deleted unnecessary CSS files.

### CODE REVIEW

File: filters/tatari/sandbox/sandbox.jsx [hunk 1]
```diff
@@ -14,58 +14,123 @@ mockApi
...
-      { key: 1119509, value: 'Elizabeth Cannon' }
+      { key: 2659261, value: 'Newfoundland' },
+      { key: 1333393, value: 'Bernese Mountain Dog' },
+      { key: 9617612, value: 'Greater Swiss Mountain Dog' },
+      { key: 1670106, value: 'Saint Bernard' },
+      { key: 6286295, value: 'Great Pyrennes' },
+      { key: 8443652, value: 'Leonberger' },
+      { key: 6767534, value: 'Chinook' },
+      { key: 6310679, value: 'Kuvasz' },
+      { key: 4429528, value: 'Tibetan Mastiff' }
```
Comment (ben-burlingham) [inline filters/tatari/sandbox/sandbox.jsx]: This is cool!

Can we make sure that there are wrapping values for group, children, and ungrouped items?

Nice haircut!

/compliment sandwich

File: filters/tatari/src/Tatari.scss [hunk 2]
```diff
@@ -1,97 +1,155 @@
...
-.dropdownContainer {
-  box-sizing: border-box;
-  margin-bottom: 5px;
-  margin-right: 10px;
-  opacity: 1;
+.activeControls {
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+  padding: $p 2 * $p;
```
Comment (ben-burlingham) [inline filters/tatari/src/Tatari.scss]: Recommend `(2 * $p)` for clarity

File: filters/tatari/sandbox/sandbox.jsx (hunk 1 above)
Comment (nickwmurphy) [inline filters/tatari/sandbox/sandbox.jsx]: 👍

File: filters/tatari/src/Tatari.scss (hunk 2 above)
Comment (nickwmurphy) [inline filters/tatari/src/Tatari.scss]: 👍

**Pull request merged**